config/initializers/new_framework_defaults.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Be sure to restart your server when you modify this file.
+#
+# This file contains migration options to ease your Rails 5.0 upgrade.
+#
+# Once upgraded flip defaults one by one to migrate to the new default.
+#
+# Read the Rails 5.0 release notes for more info on each option.
+
+# Enable per-form CSRF tokens. Previous versions had false.
+Rails.application.config.action_controller.per_form_csrf_tokens = true
```

Comment:
Thanks for reading that careful (I might have overlooked that in another PR), I just wondered myself and double-checked that. According to the Guid on the Ruby on Rails website [1], this is correct (and it doesn't throw an error or exception)

[1]: http://edgeguides.rubyonrails.org/configuring.html#configuring-action-controller